gitkraken/PKGBUILD

```diff
@@ -16,7 +17,7 @@ url="https://www.gitkraken.com/"
 provides=('gitkraken')
 arch=('x86_64')
 license=('custom')
-depends=('nss' 'libxtst' 'org.freedesktop.secrets' 'alsa-lib' 'libxss')
```

Comment:
Can you add this as `optdepends` because the Debian Package offers this as a suggestion package?